src/administration/vouchers.md

```diff
@@ -0,0 +1,13 @@
+# Vouchers 
+
+## Applying a voucher to your project
+
+If you receive a [Platform.sh](https://platform.sh/) voucher you can redeem it as follows:
+
+* Go to your account settings via the UI link in the top-right corner of the screen (where your profile picture is) or using this link: [https://accounts.platform.sh/user](https://accounts.platform.sh/user).
+* In the left navigation click on vouchers (the seventh entry)
+* On the page click on "add a voucher code"
+* Enter the code and click on the "ADD CODE" button
+* Et voila! Your account will now be credited with additional dollar, pounds or euros. 
+
+If you are assessing Platform.sh for your organziation and think that you could benefit from a little more oomf in your test project why not contact us to request a voucher? Write to: sales@platform.sh.
```

Comment:
Should this be encoded somehow so that it's not quite so easy to spam harvest?  Or is that already a lost battle?